src/app/components/PageLayout/ScrollingCard.tsx

```diff
@@ -0,0 +1,20 @@
+import Card from '@mui/material/Card'
+import { FC, useEffect, useRef } from 'react'
+import { useLocation } from 'react-router-dom'
+
+export const ScrollingCard: FC<Parameters<typeof Card>[0]> = props => {
```

Comment:
add some documentation why this is needed

Reply:
OK, added documentation.